metecho/api/serializers.py

```diff
@@ -117,6 +118,20 @@ class Meta:
         fields = ("id", "username", "avatar_url")
 
 
+class GitHubOrganizationSerializer(serializers.ModelSerializer):
+    id = serializers.CharField(read_only=True)
+
+    class Meta:
+        model = GitHubOrganization
+        fields = ("id", "name", "avatar_url", "members", "currently_refreshing")
```

Comment:
We shouldn't expose the org's members to the frontend.

Reply:
Related to https://github.com/oddbird/Metecho/pull/1376#discussion_r687856137